### ISSUE TITLE
Find a bijection with chained parser

### ISSUE THREAD

**Issue by nicolas-zozol** [d-plaindoux/masala-parser#111]
To find the offset, I need to use: (`parser_chain_test.js`)

```js
response.input.source.offsets[response.offset]
```

which is neither pretty or safe.

If there are different level, it would be nice to have a recursive function that goes to the initial StringStream to find the error offset.

**Comment by nicolas-zozol** [comment]
It's already the case. In `ParserStream`:

```js
   location(index) {
        return this.input.location(this.getOffset(index - 1) + 1);
    }
```
The literal function of genlex gives:

`response.reject(input.location(index), false)`

So it should give back the child offset using: `this.offsets[index]`

```
getOffset(index) {
        // FIX ME: not safe; probably this.offsets[index] || 0
        return this.offsets[index] || index; 
}
```

**Comment by nicolas-zozol** [comment]
Closed with #110
